cookbook/sandbox.config

```diff
@@ -4,11 +4,14 @@ python_venv=flytekit_venv
 
 [auth]
 assumable_iam_role=arn:aws:iam::111234567892:role/yourrole-development
+raw_output_data_prefix=s3://my-s3-bucket/offloaded-data-storage
 
 [platform]
 url=host.docker.internal:30081
 insecure=True
 
+# Deprecated. Please use the raw_output_data_prefix above
```

Comment:
I think we should just delete it?

Reply:
If we want to delete the current config objects, then I need to add more code to the [flytekit pr](https://github.com/lyft/flytekit/pull/167).  Specifically, the new raw output config object is not currently used at run time, it’s only used at compile time (sent up to admin at launch plan registration).

Currently in that PR, if it's missing at runtime, it'll fall back to the current aws_shard/gcs_prefix config objects.  If we want to remove the old configs entirely, we'll have to make it so that if Admin doesn't send it (because users are on an older version of admin), we do our own sharding based off of the raw output config at runtime, when click starts up for instance. 

However, if users update flytekit, but don't add a raw output config _and_ we remove the existing config objects, then at runtime, it won't know where to write to.

So we can,
1) We can just add runtime deprecation notices and delete them later.
2) Add a fallback at registration time to the existing shard formatter/prefix if the raw output prefix is missing.

I actually think I'd go with option 1 for now, at least until the different sharding methods you just added to flyteplugins are available for picking and choosing.  I hate assuming migration timelines though.

Reply:
I meant delete it in the example? Set good precedence

Reply:
talked offline, I understand the reason is for backwards compatibility - Someone may run the example on an older Flyte installation.